platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -32,10 +32,21 @@ MEMORY
   CODE_RAM (rwx) : ORIGIN = S_CODE_SRAM_ALIAS_BASE, LENGTH = TOTAL_CODE_SRAM_SIZE
 #endif
 #ifndef TFM_MULTI_CORE_TOPOLOGY
+/* Define CMSE_VENEER_REGION_IN_CODE to place the veneers in FLASH together with
+ * the rest of the code. If so, also define CMSE_VENEER_REGION_START_ALIGN and
+ * CMSE_VENEER_REGION_END_ALIGN.
+ */
+#ifndef CMSE_VENEER_REGION_IN_CODE
   VENEERS  (rx)  : ORIGIN = CMSE_VENEER_REGION_START, LENGTH = CMSE_VENEER_REGION_SIZE
 #endif
+#endif
 }
 
+#ifndef CMSE_VENEER_REGION_IN_CODE
```

Comment:
@oyvindronningstad strictly speaking, this is only needed insind an #ifndef TFM_MULTI_CORE_TOPOLOGY

Reply:
Added an #ifdef TFM_MULTI_CORE_TOPOLOGY around it.